components/asana/actions/add-task-to-section/add-task-to-section.mjs

```diff
@@ -1,40 +1,89 @@
 // legacy_hash_id: a_njiaY8
+import asana from "../../asana.app.mjs";
 import { axios } from "@pipedream/platform";
```

Comment:
Since you are refactoring the code here and in every other component, I would suggest also to use `axios` lib only in `asana.app.mjs` file and in there define all the functions that will deal with `axios` requests. In that way we can remove all `axios` imports on each component